src/main/java/com/codejoust/main/dto/game/GameMapper.java

```diff
@@ -84,9 +85,22 @@ public static SubmissionDto submissionToDto(Submission submission) {
             return null;
         }
 
+        List<SubmissionResultDto> notHidden = new LinkedList<>();
```

Comment:
Update the `submissionToDto` mapper test within the [`GameMapperTests.java`](https://github.com/CodeJoustHQ/main/blob/3f517ab1ed3a151e46b63a01012bf9bd663323c6/src/test/java/com/codejoust/main/mapper/GameMapperTests.java#L136-L154) file to test this new functionality.

If possible, I think it'd also be helpful to add a comment header for this method (or within it) explaining that this filters out all hidden test results.